lib/elixir/lib/enum.ex

```diff
@@ -2046,6 +2046,24 @@ defmodule Enum do
   end
 
   @doc """
+  Sorts the collection, removing all duplicated items.
+
+  ## Examples
+
+      iex> Enum.usort([5, 1, 2, 3, 2, 1])
+      [1, 2, 3, 5]
+
+  """
+  @spec usort(t) :: list
+  def usort(collection) when is_list(collection) do
+    :lists.usort(collection)
+  end
+
+  def usort(collection) do
+    collection |> uniq |> sort
```

Comment:
Wouldn't `collection |> to_list |> :lists.usort` work just as well as `|> sort |> sorted_uniq` and have the same complexity?